homeassistant/components/nest/climate_sdm.py

```diff
@@ -255,16 +251,17 @@ def preset_modes(self):
     def fan_mode(self):
         """Return the current fan mode."""
         if FanTrait.NAME in self._device.traits:
-            trait = self._device.traits[FanTrait.NAME]
+            trait = self._device.traits.get(FanTrait.NAME, {})
```

Comment:
The line above already ensures it exists...?

Reply:
Indeed, this is a mistake I should have reverted. Thanks.